### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Get Shared Access Signature_Use of Color : The "Learn More about shared access signature" are not associated with underline/chevron to differentiate as link from normal text on dialog.

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/AzureStorageExplorer#7583]
### Preflight Checklist

- [X] I have installed the [latest version of Storage Explorer](https://github.com/Microsoft/AzureStorageExplorer/releases/latest).
- [X] I have checked existing resources, including the [troubleshooting guide](https://docs.microsoft.com/azure/storage/common/storage-explorer-troubleshooting) and the [release notes](https://github.com/microsoft/AzureStorageExplorer/releases).
- [X] I have searched for [similar issues](https://github.com/Microsoft/AzureStorageExplorer/issues).

### Storage Explorer Version

1.32.1 (96)

### Architecture

x64

### Storage Explorer Build Number

20231114.10

### Platform

Windows

### OS Version

Windows 11 Enterprise Insider Preview (23H2)

### Bug Description

GitHub Tags:
#A11yMAS;#A11yTCS;#Win11;#DesktopApp;#A11ySev3;#BM_AzureToolsStorageExplorer_Win32_Dec2023;#Azure Tools Storage Explorer;#WCAG1.4.1;#UseofColor;

Environment Details:
Application: Microsoft Azure Storage Explore
version: 1.32.1 (96)

### Steps to Reproduce

1. Launch Storage Explorer
2. Tab till "Open Connect dialog" pane and Sign in to Azure.
3. Tab till "Manage Accounts" pane and Select the subscriptions you will use. Once selected, context menu on "Apply" button.
4. Tab till "Blob Containers" of the selected subscription, press context menu and hit enter on "Create Blob container" to Create a new Blob container.
5. Once completed, Tab till "Upload" button and Upload a blob/file by using "Upload Files" control.
6. Blob gets uploaded, press context menu button on blob and select "Get Shared Access Signature" control and hit enter.
7. Verify that the "Learn More about shared access signature" are associated with underline/chevron to differentiate as link from normal text on dialog or not. 

### Actual Experience

The "Learn More about shared access signature" are not associated with underline/chevron to differentiate as link from normal text on dialog.
**Note:** Similar issue is observed in "Manage Access policies" scenario also.

### Expected Experience

The links on the page should have the underline or chevron associated with it, in-order to differentiate from normal text on page.

### User Impact: 
Color blind users will face difficulties to identify and easily distinguish the links on page from normal text as it does not have any underline/chevron associated with it.

### Attachment:
![A11y_Azure Tools Storage Explorer_Get Shared Access Signature_Use of Color ](https://github.com/microsoft/AzureStorageExplorer/assets/98530069/9386ef63-1412-4935-80f4-5a7cddaf1b97)
![Manage Access policies_Useof Color](https://github.com/microsoft/AzureStorageExplorer/assets/98530069/636a6fbb-b6c5-4f98-b444-3cd5747ef3c3)

**Comment by JasonYeMSFT** [comment]
Need to update ui-components and any packages using it for dialogs.

**Comment by kupatkar99** [comment]
Verified the issue in below environment:
Microsoft Azure Storage Explorer
version: 1.33.0-dev(96)
Issue is not fixed. Please find below added attachment for reference.
![7583_not fixed](https://github.com/microsoft/AzureStorageExplorer/assets/98530069/04b81c0c-584e-46eb-aad7-52a8b72eb869)

**Comment by JasonYeMSFT** [comment]
I added the underline on main.

**Comment by kupatkar99** [comment]
Verified the issue in below environment:
Microsoft Azure Storage Explorer
version: 1.33.0-dev(96)
Issue is fixed. Hence Closing the bug please find added attachment for reference.
![7583_Fixed- shared access](https://github.com/microsoft/AzureStorageExplorer/assets/98530069/9cdd1077-8a7b-4548-a140-a44815e895df)
![7583_Fixed- Access policy](https://github.com/microsoft/AzureStorageExplorer/assets/98530069/79a4ab29-138f-4a9c-b06b-0db7142c88f8)
